src/sources/Mvt.js

```diff
@@ -68,7 +94,11 @@ export default class MVT extends Base {
     }
 
     requestData(viewport) {
-        return this._tileClient.requestData(viewport, this.responseToDataframeTransformer.bind(this));
+        return this._tileClient.requestData(viewport, this.responseToDataframeTransformer.bind(this),
+            zoom => this._options.maxZoom == undefined ?
```

Comment:
I'd create a function in the MVT object like `responseToDataframeTransformer`

Reply:
I don't find that clearer